src/libs/actions/Session.js

```diff
@@ -143,9 +143,10 @@ function fetchAccountDetails(login) {
  * re-authenticating after an authToken expires.
  *
  * @param {String} authToken
+ * @param {String} encryptedAuthToken
  * @param {String} email
  */
-function createTemporaryLogin(authToken, email) {
+function createTemporaryLogin(authToken, encryptedAuthToken, email) {
```

Comment:
I don't think we need encrypted auth token here, right? it won't be returned from CreateLogin (or SetPassword) according to the web PR

Reply:
Well, it's not returned from `CreateLogin` but it is returned by `SetPassword` (see [here](https://github.com/Expensify/Web-Expensify/blob/56761557c21e6fcc5add90bbb5cff1758771ffd2/lib/UserAPI.php#L2515)).

The reason it's an argument to this function is because this is the only place `setSuccessfulSignInData` is called from. i.e: only after `CreateLogin` succeeds is the user actually signed in, and that's when the `authToken` is saved in Onyx. So it makes sense that the `encryptedAuthToken` would be set at the same time.